lasertagger/bert/tokenization.py

```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
 class FullTokenizer(object):
   """Runs end-to-end tokenziation."""
 
-  def __init__(self, vocab_file, do_lower_case=True):
+  SPECIAL_SYMBOL_LIST = [".", ",", ";", "!", "?", ":", 
+                               "##.", "##,", "##;", "##!", "##?", "##:"]
+  
+  def __init__(self, vocab_file, do_lower_case=True, do_masking=False):
     self.vocab = load_vocab(vocab_file)
     self.inv_vocab = {v: k for k, v in self.vocab.items()}
```

Comment:
what does inv_vocab mean? use more meaningful name

Reply:
This is part of the original bert code, and I do not intend to use this part.